accepted/0000-install-peer-deps.md

```diff
@@ -0,0 +1,144 @@
+# Install Peer Dependencies
+
+## Summary
+
+Install `peerDependencies` along with packages that peer-depend on them.
```

Comment:
what happens if the peerDep is installed but doesn't fit within the range? I may be missing it somewhere in the `Detailed Explanation` section.

Reply:
Then that's an error.  (More in next comment.)